### ISSUE TITLE
abrt-applet crashes after clicking restart in notification

### ISSUE THREAD

**Issue by mphojele** [abrt/abrt#1602]
OS: Fedora Linux 36 (Workstation Edition Prerelease)
abrt: Version 2.15.1

I was reproducing an issue(https://gitlab.gnome.org/GNOME/yelp/-/issues/191) and the notification to say the program crashed popped up and on clicking restart nothing happened. In the logs I found that abrt-applet dumped core.

Another interesting thing is that when the applet crashes it is not reported and there's a message in the logs as seen in the attached screenshot.
![abrt-applet_crash_ignored](https://user-images.githubusercontent.com/44401412/160083198-076e0e12-aecf-406a-92f9-e20b0da8365a.png)

## Core dump file
[core.abrt-applet.zip](https://github.com/abrt/abrt/files/8348695/core.abrt-applet.zip)

**Comment by msrb** [comment]
Yeah, ABRT currently ignores its own crashes to prevent infinite crash-report-crash loops. However, `abrt-applet` could be excluded from this ignore list for sure.
